README.md

```diff
@@ -58,10 +43,25 @@ meson build --prefix=/usr
 cd build
 ninja && sudo ninja install
 ```
-### Questions and issues
-If you want to ask any question about the project please refer to the [Forum Thread](https://forum.alecaddd.com/d/50-akira-development).
 
-If you are trying out Akira and you encounter an error or any problem feel free to just open an issue
+## Questions and issues
+If you want to ask any question about the project, we have a dedicated Discord channel available to any [Patreon](https://www.patreon.com/akiraux) supporter.
+
+If you are trying out Akira and you encounter an error or any problem feel free to just open an issue.
+
+## Donations
+If you like Akira and you want to support its development, consider donating via [Liberpay](https://liberapay.com/AkiraUX/) or pledge on [Patreon](https://www.patreon.com/akiraux).
+
+### Official Mascot
+![](akira-mascot-akari.png)
```

Comment:
We should use the image from the assets repository and remove it from the source repo.

Reply:
Updated